homeassistant/components/fritz/sensor.py

```diff
@@ -170,7 +170,6 @@ class FritzSensorEntityDescription(SensorEntityDescription, FritzRequireKeysMixi
         device_class=SensorDeviceClass.TIMESTAMP,
         entity_category=EntityCategory.DIAGNOSTIC,
         value_fn=_retrieve_device_uptime_state,
-        is_suitable=lambda info: info.mesh_role != MeshRoles.NONE,
```

Comment:
this will disable the sensor for every `wan_enabled = False` devices, since it is the default suitable check
with the following, the sensor is always suitable:
```python
is_suitable=lambda info: True
```